roles/openshift_storage_glusterfs/tasks/glusterfs_registry.yml

```diff
@@ -9,6 +9,20 @@
     glusterfs_storageclass: "{{ openshift_storage_glusterfs_registry_storageclass }}"
     glusterfs_image: "{{ openshift_storage_glusterfs_registry_image }}"
     glusterfs_version: "{{ openshift_storage_glusterfs_registry_version }}"
+    glusterfs_block_deploy: "{{ openshift_storage_glusterfs_registry_block_deploy | bool }}"
```

Comment:
Why are we setting all of these facts?

Reply:
Both `glusterfs_config.yml` and `glusterfs_registry.yml` include `glusterfs_common.yml` and they are mapping their specific variables to the common variables. `config` vars apply to the `glusterfs` inventory group and `registry` vars apply to the `glusterfs_registry` group.

Reply:
We don't need set_fact for this.  We can pass in items to the included task file like this:

```yaml
include: common.yml
vars:
  glusterfs_block_image: "{{ openshift_storage_glusterfs_registry_block_image }}"
```

This will prevent any collisions with any roles/plays a user might add.  We should use set_fact as little as possible.

Reply:
Okay, but that is the pattern I followed from other playbooks and is the current pattern here. I don't want to start the change in implementation for just these specific variables and I consider it out of scope to do so in this PR. I would ask that you allow this if only on precedent and then raise an Issue to set them properly later.

Context: This PR needs to land in 3.7. :)